cluster/manifests/external-dns/deployment.yaml

```diff
@@ -35,13 +35,15 @@ spec:
         - --txt-owner-id={{ .Region }}:{{ .LocalID }}
         resources:
           limits:
-            memory: 100Mi
-          requests:
             cpu: 25m
             memory: 100Mi
+        livenessProbe:
```

Comment:
any reason why a livenessProbe is used?
I think it needs some more reasoning, not to drop it, but why we use it.

Reply:
Yes, we should align on when to use what. I see `livenessProbe` as "damn, something is  wrong" and `readinessProbe` as "I'm fine but my caches aren't warm so please let me alone".

Since `/healthz` of ExternalDNS is [dead simple](https://github.com/kubernetes-incubator/external-dns/blob/master/main.go#L254) something clearly wrong must be going on.

Reply:
@szuecs does it make sense?

Reply:
fine for me